Add <SubNav> and <SubNavItem> components

paulocfjunior opened the pull request:
![image](https://user-images.githubusercontent.com/38889364/70162414-61382480-169c-11ea-9c4e-2ce74977735c.png)

matthieuh commented on src/components/SubNav.tsx:
```diff
@@ -0,0 +1,87 @@
+export const SubNav = styled('nav')<SubNavProps>(
+  css({
+    height: 48,
+    backgroundColor: 'light',
```
`light` is not a color existing in our theme

ldez commented:
@paulocfjunior you don't need to "merge master" unless you have conflicts to manage: the bot update the PR before the merge.

FYI to merge a PR, after the PR was approved, you have to add the label `status/3-needs-merge` and the bot will manage the rebase of the PR and the merge.

paulocfjunior commented:
BTW, @ldez, I've fixed the point that Matthieu mentioned and also changed the `<SubNavItem>` component to have an `<a>` as primitive (instead of a button, as is in `<Nav>`), because I noticed that we usually use it as a link to other routes. Does it make sense?

matthieuh commented:
> BTW, @ldez, I've fixed the point that Matthieu mentioned and also changed the `<SubNavItem>` component to have an `<a>` as primitive (instead of a button, as is in `<Nav>`), because I noticed that we usually use it as a link to other routes. Does it make sense?

I add the ability to change primitive in the Nav component. Do you think that it could possible to handle it there as well ? https://github.com/containous/faency/pull/4/commits/4e860e9099b38bd232148a9268ffbe5b3f44e1cc

matthieuh commented on src/components/SubNav.tsx:
```diff
@@ -0,0 +1,81 @@
+    '&::-moz-focus-inner': {
+      border: 0,
+    },
+    '&:hover': {
```
I found that we miss to set hover styling when the item is active. I handled it in the Nav component https://github.com/containous/faency/pull/4/commits/9c661b8e3759e27c189f7543b7acda452d173902

paulocfjunior commented:
@matthieuh, it's ready for a final review and approval :heavy_check_mark:
